MaxText/tests/forward_pass_logit_checker.py

```diff
@@ -111,6 +111,11 @@ def main(config, test_args):
     max_logging.log(f"{full_train_logits[0, 0, :]=}")
     token_size = int(test_args.token_size) if test_args.token_size else golden_logits.shape[0]
     max_logging.log(f"Max Difference {np.max(np.subtract(full_train_logits[0, :token_size, :], golden_logits[:token_size, :]))}")
+    aerr = np.linalg.norm(np.subtract(full_train_logits[0, :token_size, :], golden_logits[:token_size, :]))
```

Comment:
We can use `np.testing.assert_allclose` as well.
https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_allclose.html

It will print the difference once failing the tests:
```
np.testing.assert_allclose(np.sin(x), y, atol=1e-15, strict=True)
Traceback (most recent call last):
    ...
AssertionError:
Not equal to tolerance rtol=1e-07, atol=1e-15

(dtypes float64, float32 mismatch)
 ACTUAL: array([ 0.000000e+00,  1.224647e-16, -2.449294e-16])
 DESIRED: array([0., 0., 0.], dtype=float32)
```